File: 1291-sequential-digits/1291-sequential-digits.py
class Solution:
    def sequentialDigits(self, low: int, high: int) -> List[int]:

        result=[]
        number=1
        n=1
        while number!=9:
            if n==10:
                n=1
                number=int(str(number)[0])+1
            if str(number)[-1]!=str(n-1):
                n+=1
            else:
                number=number*10+n
                if low<=number<=high:
                    result.append(number)

        return(sorted(result))

File: 1291-sequential-digits/1291-sequential-digits.java
class Solution {
    public List<Integer> sequentialDigits(int low, int high) {


             ArrayList<Integer> result=new ArrayList<Integer>();
        result.sort(null);
        for(int i=1;i<10;i++){
            int tmp=i;
            int nextDigit=i+1;
            while (tmp<high+1 && nextDigit<10){
                
                tmp=tmp*10+nextDigit;
                if (low-1<tmp && tmp<high+1) {
                    result.add(tmp);
                }
            nextDigit+=1;
            }

        }
    result.sort(null);
    return result;
    }
}